### PR TITLE
Fixes#394 Update DockerFile

### LINKED ISSUE #394: Local deployment using Docker is not working.
I was setting up the local deployment of badgeyay using Docker. It gives error whenever `RUN pip install --no-cache-dir -r requirements.txt` line comes. 

`ERROR: pip not found
The command '/bin/sh -c pip install --no-cache-dir -r requirements.txt' returned a non-zero code: 127`

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #394 
Previous Results 
image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Update Dockerfile

image

### CODE REVIEW

**Comment by ParthS007** [comment]
@harshit98 @djmgit @vaibhavsingh97 Please Review

File: docker.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+gunicorn==19.7.1
```
Comment (vaibhavsingh97) [inline docker.txt]: Why to create new file, you could reuse `requirements.txt`

Comment (ParthS007) [reply docker.txt]: `Dockerfile` was giving error while executing when I used `requirements.txt`, that's why I have added `docker.txt`.

Comment (ParthS007) [reply docker.txt]: Now I have made the changes @vaibhavsingh97 , I have to revert all the changes for reproducing it.

**Comment by harshit98** [comment]
@vaibhavsingh97 Even I tried.
 
`COPY requirements.txt /app/requirements.txt` I don't know why docker is not running this command properly. The directory address is fine though.  @ParthS007 Change the `docker.txt` to some other meaningful name.

Also, try another method. Define the directory, which I have mentioned above. Then restart the docker service by using `sudo service docker restart` and again start docker build by using `docker build badgeyay:latest .` 

There are some issues with the new version of docker. For some devs, restarting the docker service worked.

Source: <a href="https://stackoverflow.com/questions/32997269/copying-a-file-in-a-dockerfile-no-such-file-or-directory">COPY command error even after directory is correctly defined</a>.

**Comment by ParthS007** [comment]
@harshit98 I have changed the file name. 👍

**Comment by vaibhavsingh97** [comment]
@ParthS007 Change you Dockerfile to this and drop `docker.txt`
```
FROM python:3.6

# We copy just the requirements.txt first to leverage Docker cache
COPY ./app/requirements.txt /app/

WORKDIR /app

RUN pip install -r requirements.txt

COPY . /app

ENTRYPOINT [ "python" ]

CMD [ "main.py" ]

```

**Comment by ParthS007** [comment]
@harshit98 @vaibhavsingh97 Please Review

File: requirements.txt [hunk 2]
```diff
@@ -1 +1 @@
--r requirements/prod.txt
+--r requirements/prod.txt
```
Comment (vaibhavsingh97) [inline requirements.txt]: It should be `-r` instead of `--r`

Comment (vaibhavsingh97) [reply requirements.txt]: Can you tell me any reason for this?

Comment (ParthS007) [reply requirements.txt]: @vaibhavsingh97 It is as it is , I haven't made changes in it.

**Comment by ParthS007** [comment]
@vaibhavsingh97 I think, Now everything is fine

**Pull request merged**